---
layout: post
title: 'Simple GRN'
date: 2020-05-17
thumbnail: thumbnail.png
dependencies:
---

# Infection

{% raw %}

<script>
function resizeIframe(obj) {
obj.style.height = obj.contentWindow.document.body.scrollHeight + 'px';
obj.style.width = obj.contentWindow.document.body.scrollWidth + 'px';
}
</script>
<iframe frameborder="0" marginheight="20" marginwidth="35" scrolling="no" onload="resizeIframe(this)" src="js//index.html"></iframe>
{% endraw %}

# Snell-Rood_2010

| **Title:**   | **Toward a population genetic framework of developmental evolution: the costs, limits, and consequences of phenotypic plasticity** |
| ------------ | ---------------------------------------------------------------------------------------------------------------------------------- |
| **Authors:** | **Emilie C. Snell-Rood** , **James David Van Dyken**, **Tami Cruickshank**, **Michael J. Wade**, and<br>**Armin P. Moczek**        |
| **Year:**    | 2010                                                                                                                               |
| **Tags:**    | #plasticity, #modularity, #development, #drift, #GRN                                                                               |


**Summary:**
A modular GRN can increase evolvability and facilitate the evolution of plasticity and environment-specific gene expression to multiple different environments through the reduction of **pleiotropic effects**, so that expression patterns can evolve independently. However, in a sub-divided population where only a subset of individuals experience a certain environment **selection is reduced** and thus deleterious mutations are more likely to fix while beneficial mutations can be lost because of **increased drift** as the Ne of the trait is decreased. This can result in loss of plasticity due to the increased **mutational load**. On the other hand, this weekend selection can increase the **cryptic genetic variation** in the population that can fuel evolution when the environment switches back. 

**Interesting because:**
Experimental evidence of:

- Bacteria can survive across a **greater range of environments have more modular** metabolic networks than those that are limited to less variable environments 
- 5â€“50% of the genome may vary in expression with environment
- many genes are expressed across environments, but at different levels
- list of growing support for the prediction that environment-specific genes should accumulate variation within species
- experimental evolution where **adaptation occurs more rapidly in specialists** evolving in constant environments than in generalists evolving in fluctuating

ðŸ’¥ 
lack of expression of a focal gene in one environment may only be possible through the induced expression of an **inhibitory transcription factor** recognizing and blocking a specific regulatory site. While the **protein-coding portion** of our focal gene experiences **relaxed** selection, the transcription factor and **regulatory site do not.**

ðŸ’¥
relaxed selection may therefore vary substantially with **network architecture**

#research_idea
Different network architectures have different evolvability because of this? Not just modularity enabled reduced pleiotropy




![](https://paper-attachments.dropbox.com/s_83D6EE6033C5BE25939BEE0E370DABC74EF573F48E65A5A300F1765F2280C35B_1599079680123_image.png)




**The link to the paper:**


https://www.dropbox.com/s/cakrs8gv7k5tifu/Snell-rood%20et%20al._2010.pdf?dl=0


